RadixWallet/Clients/CacheClient/CacheClient+Interface.swift

```diff
@@ -200,6 +198,8 @@ extension CacheClient {
 			case .dateOfFirstTransaction:
 				"DateOfFirstTransaction"
```

Comment:
Maybe take the opportunity to turn this into a local constant?

Reply:
hopefully translated into Sargon soon :)